### DISCUSSION TITLE
Lower Monika's language intros

### DISCUSSION THREAD

**Discussion by Finikyu** [Monika-After-Story/MonikaModDev#7856] [issue discussions]
I personally believe that the amount of languages Monika is trying out, to the point of speaking it without assistance even if not fluently, is too high.
She has Japanese, Latin, French, Esperanto, Italian, including English most people only learn 2-3 languages...it's way too much and honestly comes off as Mary-Sueish even if she's not perfect....
I'd like to recommend removing some of these as honestly, it's too much. Personally I'd keep Japanese and Latin, based on her origins and interest in language, but the others are too much really.

**Comment by 20RaVen23** [comment]
![sniff](https://user-images.githubusercontent.com/55263957/119246148-230f1900-bb4d-11eb-99b1-0bfd5175e970.png)

**Comment by Retrolovania** [comment]
She's only stated that she's practicing them. As of now, it's yet to become clear which ones she's working on more.
Plus, trying to memorize several languages isn't very unrealistic. Lots of people have done it, actually. So, in that regard, I don't see much unrealism with her wanting to broaden her linguistic horizons.

**Comment by Finikyu** [comment]
I know she says that, but who practices 6 languages at once?
It's crazy and unrealistic, you say lots of people have done it but it's an incredibly rare skill, and most of them are bi-lingual by upbringing.
If you think practice six languages is normal then I don't know what to say. :S

**Comment by Retrolovania** [comment]
_shrug_
People have done it, regardless. I believe the term used is "polyglot."
It may be rare in terms of how often it shows up, but given the over 7.5 billion people on the planet, that's still a lot of people with that trait.
Plus, again, she's not explicitly trying to master them; she's just practicing them. Who knows? She might not even be serious about learning them in full; only some. In my opinion, she's the kind of person who's eager to try out new things if they interest her enough.

**Comment by Wingdinggaster656** [comment]
Monika is never a normal person at first.
Do you think a person who stay on top of everything is normal? This kind of person is always less the 1%.
Not to mention she is a huge fan of literature.
So I personally disagree.

**Comment by Rai99** [comment]
I can agree that it's a bit too much, and unless you really have a thing for multiple languages, I think it kinda loses its charm after the first few. Japanese and Latin make sense given her origins and interests, but I do feel like the rest kind of come out of nowhere.

**Comment by Finikyu** [comment]
Who practices more than one new language at a time? Well enough to not need a guide to do so, and for what purpose? 
I wouldn't mind a common greeting like "Bonjour" because it's common and most people know it but to be actively practicing suggests some kind of end, rather than just idle dabbling.
Hence why having six is wayyy too much, she's supposed to be normal not a super-genius. And most only know 1, 2 is they grew up in a bi-lingual household. 6 even casually is too much for my suspension of disbelief and I bet potentially others too.

**Comment by Rai99** [comment]
Even if Monika does enjoy literature, does that really involve learning different languages? You can like and enjoy literature, but I think the matter of completely different languages is a different question.

**Comment by Retrolovania** [comment]
There is the possibility that, even if the dialouge box has her saying the words correctly, the pronunciation may not be correct if she was able to speak through voice. So, even if she was exceptionally good at doing it through text, through voice is a completely different story.
I'll be honest, I can't really see a major issue with all this.

**Comment by Finikyu** [comment]
> Monika is never a normal person at first.
> Do you think a person who stay on top of everything is normal? This kind of person is always less the 1%.
> Not to mention she is a huge fan of literature.
> So I personally disagree.

She believes herself to be, her story before the epiphany was supposedly normal and having a strange background isn't grounds for knowing more than the average person ever would.
Also to be honest being a fan of literature doesn't mean you'd necessarily have an interest in languages outside of its influence in literature. It's more reasonable to appreciate literature in your native language than it's original, like noone reads Hunchback of Notre Dame in French if they don't already know it, they'd just read it in English.

**Comment by Finikyu** [comment]
It's not a major issue, but it has bothered me for some time.
Also no reason to assume she's failing at it either, especially if she's been practicing it. Kinda needing to do mental gymnastics to think she is potentially failing.

**Comment by Retrolovania** [comment]
Knowing different languages can help people read and watch content that the version done in their native language may not have translated properly, or could have left out some details in the process. Monika may find an interest in reading material in its original form, and possibly compare it to the English (assuming that's her first language) version to see what had been altered.
Kinda like how she talks about comparing books to their film adaptations, except for this, it's the same book told in different languages.

**Comment by Finikyu** [comment]
Noone is arguing that knowing different languages isn't incredibly helpful, I'm just saying the extrent to how many she's invested in is too much, I believe it's beyond reasonability.

**Comment by MaximumRustler** [comment]
I'd also agree with this issue. To me, practicing would imply actively learning them, which I don't see making much sense. Having an interest in them and looking up some basics out of curiosity to just say a line or two is fine and maybe something interesting about that language, but I wouldn't say that's practicing, if anything maybe that should be changed to not imply actively practicing. Also while some people have, it's not really a thing most can do, especially trying to learn 6 at once, that's ridiculous. Also, saying Monika isn't normal kinda goes against her and what she's said herself. Yes she's a top student, but that doesn't mean someone can easily try to learn 6 at once. I can't say any top student I've known that went out of their way to do that, at most they know 2 or sometimes 3, and the second only cause of their upbringing. Not to mention how much time that'd take, even if you had absolutely nothing else to do learning 5-6 at once is unrealistic, and in Monika's case she does have other things she does too like learning coding, so not sure where she'd get the time even if that was the only issue. Also, of course knowing other languages is helpful, but most people would focus on one or two that they'd probably use most. I don't see interest in literature being a reason to learn that many, can't say I've heard of it being much of a thing that people go out of their way to learn the language of every thing they're interested in literature. Would more be a thing if that person is interested in going into that field, but even then I can only see one or two.

**Comment by lunulae** [comment]
I'd personally prefer to solve this in one of two ways.
First, maybe to rework the language greetings that specify she's practicing that language, so that each player can make their own assumptions about which ones she's learning actively and which ones she's just learning a few phrases in.

The second might be to convert these to a submod or otherwise add a level of personalization in-game--a lot of people have enjoyed having Monika speak their language, and it might be good to preserve that and make it feel special while cutting down on the overall number of language greetings each player would get. But it would take more work to get it done, so it might be worth doing an interest check to see if it's worth the resources to go this route.

**Comment by ThePotatoGuy** [comment]
I have similar sentiments with having her greet in too many different languages. I think a more interesting approach could be to ask the player what languages they know, and if they do know another language, then only show that language's practice greeting in the future, with JP as the default if the player doesn't know another language. So while Monika will have multiple language greetings, only 1 or 2 will ever actually be shown to a specific player.

**Reply by Justformas** [reply]
While I do think there might be a bit too many language greetings (and would rather or like to see some rarer greetings a little more often), I don't think it's unfathomable for Monika to at least partly know several languages. She was the top student in her class, and since then all the free time she has in MAS only makes it seem more likely that she has time to learn other languages. As for default language greetings, some might not just be good for default, but still always enabled. I think the Esperanto one makes sense for that since it's a bit different; it was created artificially and she says it makes sense it would interest her, given her background. Japanese should at least be a default but could possibly be another one that's always enabled, and maybe Latin.

Alternatively there could be personalization more along the lines of preferred or chosen language greetings to hear, rather than "I speak these languages". After all, Monika usually explains what the greetings mean in English, which wouldn't make as much sense to do if the player said they speak that language. But I like that she explains what they mean and wouldn't necessarily want to not hear some or certain language greetings just because I said I don't speak them.
